WalletWasabi.Backend/Config.cs

```diff
@@ -60,8 +56,7 @@ public async Task LoadOrCreateDefaultFileAsync()
 			AssertFilePathSet();
 
 			Network = Network.Main;
-			BitcoinRpcUser = "user";
-			BitcoinRpcPassword = "password";
+			BitcoinRpcConnectionString = "user:password";
```

Comment:
I think we should put `cookiefile=/home/user/.bitcoin/.cookie` in it, since user:password is deprecated.